src/Hl7.Fhir.Specification.Tests/Validation/BasicValidationTests.cs

```diff
@@ -1390,6 +1390,49 @@ public void ValidateWithTargetProfileAndChildDefinitions()
 
         }
 
+        [Fact]
+        public async void TestValidationPerformanceUsCore()
+        {
+            var resolver = new MultiResolver(new CachedResolver(new DirectorySource(Path.Combine("TestData", "us-core-profiles"))), new CachedResolver(new ZipSource("specification.zip")));
+
+            var ctx = new ValidationSettings()
+            {
+                ResourceResolver = resolver,
+                GenerateSnapshot = true,
```

Comment:
Generating snapshots will slow down the first validations, as the CachedResolver won't have them cached yet.

We can look into using Benchmark.net for more accurate timings.